launch/launch/main.py

```diff
@@ -22,27 +22,49 @@
 
 
 def file_exists(filename):
-    if not os.path.exists(filename) or not os.path.isfile(filename):
-        raise argparse.ArgumentError("'%s' does not exist" % filename)
-    return filename
+    return os.path.exists(filename) and os.path.isfile(filename)
```

Comment:
This check is redundant. `isfile` already implies `exists`. And in that case the function should be removed since `file_exists` maps to only `is_file`.